packages/next-codemod/transforms/__testfixtures__/next-async-request-api-dynamic-apis/async-api-15.output.js

```diff
@@ -2,7 +2,7 @@ import { cookies } from "next/headers";
 
 async function MyComponent() {
   function asyncFunction() {
-    callSomething(/* TODO: please manually await this call, if it's a server component, you can turn it to async function */
+    callSomething(/* Next.js Dynamic Async API Codemod: please manually await this call, if it's a server component, you can turn it to async function */
```

Comment:
```suggestion
    callSomething(/* Next.js Dynamic Async API Codemod: Manually await this call, if it's a Server Component. */
```

Unclear whether the if belongs to before or after comma. Seems to me, it's an async function already so no need to tell people to make it async.